### PR TITLE
WIP: VTOL timeout RTL 

### PR DESCRIPTION
**Pull request by sanderux**
This fixes the erratic behavior on transition timeout RTL
Also increased default back transition duration

### CODE REVIEW

**Comment by sanderux** [comment]
this also helps https://github.com/PX4/Firmware/pull/4522

File: src/modules/vtol_att_control/standard.cpp [hunk 1]
```diff
@@ -137,20 +137,13 @@ void Standard::update_vtol_state()
...
 
-		} else if (_vtol_schedule.flight_mode == FW_MODE) {
+		} else if (_vtol_schedule.flight_mode == FW_MODE ||
+			   _vtol_schedule.flight_mode == TRANSITION_TO_FW) {
 			// transition to mc mode
 			_vtol_schedule.flight_mode = TRANSITION_TO_MC;
 			_flag_enable_mc_motors = true;
 			_vtol_schedule.transition_start = hrt_absolute_time();
 
-		} else if (_vtol_schedule.flight_mode == TRANSITION_TO_FW) {
```
Comment (AndreasAntener) [inline src/modules/vtol_att_control/standard.cpp]: We need this, it's important should the transition go really wrong (excessive tilt/roll for some reason). Doing the normal back transition might be fatal in this case. I guess we need a soft abort for the transition timeout. I'm working on a slowdown during the back transition which could always be active in auto modes, and we remove the RTL on transition failsafe for manual modes (it doesn't make sense there anyway).

**Comment by sanderux** [comment]
On further examination it turns out you are right. I was under the assumption the MC controller had full control but was in alt-hold mode, it seems this is wrong and it actually scales the weight of the controller over the time set.

I believe this should change after which this PR will make sense again.

**Comment by sanderux** [comment]
@AndreasAntener where do you stand with velocity controlled back transition?

**Comment by sanderux** [comment]
Should be velocity or airspeed based

**Pull request closed without merging**